tests/scripts/thread-cert/border_router/nat64/test_single_border_router.py

```diff
@@ -58,6 +64,13 @@
 LARGE_NAT64_PREFIX = "ff00:00:00:01:00:00::/96"
 
 
+def SynthersizeIp6Address(ip6_network: ipaddress.IPv6Network, ip4_address: ipaddress.IPv4Address):
```

Comment:
s/SynthersizeIp6Address/synthesize_ip6_address

And move the function as a member of class Nat64SingleBorderRouter?